extension/views/regulation_pression.sql

```diff
@@ -14,6 +14,8 @@ CREATE OR REPLACE VIEW qwat_ch_vd_sire.regulation_pression AS
 		watertype_code_sire AS "Type_Eau",
 		CASE
 			WHEN installation_type = 'pump'::qwat_od.installation_type THEN 2
+			WHEN vw_export_installation2.fk_pressurecontrol_type = 2801 THEN 1
```

Comment:
vw_export_installation2 ?

Typo or custom view to fix

Reply:
Typo, fixed in the force pushed commit bellow